jmix-data/eclipselink/src/main/java/io/jmix/eclipselink/impl/dbms/SqlServerDbmsFeatures.java

```diff
@@ -52,7 +52,7 @@ public String getTransactionTimeoutStatement() {
 
     @Override
     public String getUniqueConstraintViolationPattern() {
-        return "with unique index \'(.+)\'";
+        return "(Violation of UNIQUE KEY constraint '(.+?)'|with unique index '(.+)')";
```

Comment:
This pattern doesn't work as expected. 

Please read the issue description again and compare your result with the expected one:

> ER:
Notification "A department with the same name already exists"

See docs: https://docs.jmix.io/jmix/flow-ui/exception-handlers.html#unique-constraint-violation-handler